docs/data/joy/components/alert/AlertColors.js

```diff
@@ -20,7 +20,7 @@ export default function AlertColors() {
       <Box
         sx={{
           display: 'grid',
-          gridTemplateColumns: 'repeat(3, minmax(150px, 1fr))',
+          gridTemplateColumns: 'repeat(auto-fit, minmax(150px, 1fr))',
```

Comment:
```suggestion
          gridTemplateColumns: 'repeat(auto-fit, minmax(360px, 1fr))',
```

Just bumping up their width on this demo here so they don't look too much like a button 😅

Reply:
Good call! But it overflows in smaller windows like that. How about we do this instead? This way they'll squish down to fit small screens, but still ideally look like alerts.

```suggestion
          gridTemplateColumns: 'repeat(auto-fit, minmax(150px, 400px))',
```